.gitignore

```diff
@@ -0,0 +1,4 @@
+*.swp
+template-operator
+TODO
+!vendor/**
```

Comment:
Nit: I think TODO vendor and .swp should be ignored in ones global gitignore

Reply:
We still have them in `aws-operator` which is where I took this from. No big opinion though.